### ISSUE TITLE
[CONFIG] nginx to allow bigger .gpx files

### ISSUE THREAD

**Issue by xmgz** [YunoHost-Apps/fittrackee_ynh#19]
Hi, it's not a bug but a configuration value

**TL;DR**

This is exactly my problem 

https://github.com/SamR1/FitTrackee/issues/256

*Solved* by

https://github.com/SamR1/FitTrackee/pull/257

 https://github.com/SamR1/FitTrackee/pull/257/commits/efcecb887aa5fdcb58b1b32ebde7dbc59140ec9a

set nginx configuration to allow bigger .gpx files

## Issue

I've to upload big files and changing Max. Upload Size in Admin config does not make any effect, and fittrackee server reports that the file is too big.

I guess setting ynh nginx config (for this server) to "0" would solve the issue in our fittrackee.

Thank you.

(and correct me if I'm wrong)

**Comment by Thovi98** [comment]
What's the size of your biggest file?
It's effectively correct, you can see my latest post on the forum about FitTrackee to see how to proceed in the meentime :)

**Comment by xmgz** [comment]
yes, excuse me because I had not search the forum but directly on fittrackee issues.

My problem appeared with a 3,6MB .gpx file (I have bigger ones), but if you ask me, I would set 20MB so ultrarunners could upload long distance files.

A better solution would be that nginx gets fittrackee admin value, as set in the config panel, so no conflict or confuse "too big file" message would appear. (confuse as contradiction to admin panel value)

I can wait for your fix in next updates. Thank you!
